lib/sync.js

```diff
@@ -95,6 +97,11 @@ function Sync(src, target, opts, callback) {
     };
   }
 
+  if (opts.live) {
+    this.push.on('complete', self.pull.cancel.bind(self.pull));
+    this.pull.on('complete', self.push.cancel.bind(self.push));
+  }
+
```

Comment:
This cancels the other replication in the sync, so if I `pull.cancel()` this catches it and calls `pull.cancel()`